01week/datatypes.js

```diff
@@ -0,0 +1,55 @@
+// 1. Write a JavaScript program to display the current day and time.
+const todaysDate = new Date();
+console.log(todaysDate);
+
+
+// 2. Write a JavaScript program to convert a number to a string.
+const numberConvert = 500;
+numberConvert.toString();
+
+
+// 3. Write a JavaScript program to convert a string to the number.
+Number('50');
```

Comment:
this should be a function that takes in any string and returns the string as a number